### DISCUSSION TITLE
Filtrage des statistiques régionales par secteur

### DISCUSSION THREAD

**Discussion by alemangui** [betagouv/ma-cantine#991] [Retours utilisateur·rice]
Le besoin remonté de l'usager c'est de pouvoir regarder les chiffres des cantines du secteur scolaire à l'échelle de toute la France. @JenniferStephan avait aussi remonté ce besoin.

**Comment by valeriemerle** [comment]
Autres besoins : 
cas de la région Grand Est (gérant les lycées) avoir des info sur l'ensemble des lycées publics de la région et pouvoir les comparer avec d'autres régions et avec les etablissements du secteur scolaire de sa région.
Pour une DRAAF : avoir les info pour sa région et sur chaque secteur de la RC
Pour un PAT : avoir des info à l'echelle de son PAT tous secteurs confondus ou par secteur, pouvoir se comparer avec d'autres PAT de sa région ou d'ailleurs

**Reply by alemangui** [reply]
Lié à : https://github.com/betagouv/ma-cantine/discussions/986

**Comment by hfroot** [comment]
@alemangui c'est le même filtre qui a été proposé ici ? https://github.com/betagouv/ma-cantine/issues/982

**Reply by alemangui** [reply]
Oui, mais c'est toujours assez flou ce qu'on veut faire. Une discussion sur le sujet des PATs est là : https://github.com/betagouv/ma-cantine/discussions/986
